NuRadioReco/framework/sim_station.py

```diff
@@ -31,10 +35,34 @@ def get_simulation_weight(self):
     def set_simulation_weight(self, simulation_weight):
         self.__simulation_weight = simulation_weight
 
-    def serialize(self, mode):
-        base_station_pkl = NuRadioReco.framework.base_station.BaseStation.serialize(self, mode)
+    def iter_channels(self):
```

Comment:
I don't understand this comment.

Reply:
Both the Station and the SimStation class now have a iter_channels method that does the same. So it would be better to define this once in the BaseStation class

Reply:
they are not exactly the same. The identifiers for the channels are different. We could move the data structure into the basestation and also the iter_channels function. Also the object itself is different. One is a SimChannel and the other one is a Channel. 
So I fear it might get more confusing than helping to move part of the code to the base station.

Reply:
Hmm, I see. Hard to say which solution is better. So I'm fine with either. Have you tested if Channel and SimChannel are similar enough that you can use modules like the filter and resampler on both (by just passing the SimStation instead of the Station)? I think that would be very useful

Reply:
SimChannel inherits from Channel. The only difference is that it has two more identifiers, the ray tracing and shower id. So all modules that run on the channel should also run on the simchannel. I'm actually using it in NuRadioMC .

Reply:
Great, then I don't have anything to complain about any more ;-)